WebContent/Search.jsp

```diff
@@ -26,37 +26,35 @@
 	<br/>
 	<table border="1">
 		<tr>
-			<td>Feature Name</td>
-			<td>Feature Class</td>
-			<td>State Alpha</td>
-			<td>County Name</td>
-			<td>Longitude</td>
-			<td>Latitude</td>
-			<td>Elevation (m))</td>
-			<td>Elevation (ft)</td>
-			<td>Map Name</td>
+			<td>Hotel Id</td>
+			<td>Hotel Name</td>
 			<td>Description</td>
-			<td>History</td>
-			<td>Date Created</td>
-			<td>Date Edited</td>
+			<td>Description (French)</td>
+			<td>Category</td>
+			<td>Tags</td>
+			<td>Parking Included?</td>
+			<td>Last Renovation Date</td>
+			<td>Rating</td>
+			<td>Address</td>
+			<td>Location</td>
+			<td>Number of Rooms</td>
```

Comment:
FYI, we could probably do something a lot nicer here other then number of rooms, but I feel like it would require additional web work (opening a separate page that shows the rooms' details for that hotel, for example) but I figured this was satisfactory for now given the time crunch.